client/src/pages/Login/Login.tsx

```diff
@@ -38,25 +38,18 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box
-          display="flex"
-          justifyContent="space-between"
-          alignItems="flex-start"
-          flexDirection="column"
-          className={classes.authWrapper}
-        >
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
+        <Box display="flex" alignItems="center" flexDirection="column" className={classes.authWrapper}>
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
+                  <strong>Welcome back!</strong>
```

Comment:
Alternatively, you can apply a `font-weight` using the existing CSS class

Reply:
Ya this was before I figured out how to override MUI root classes without affecting the theme. I'll address this and other changes!